torchtune/datasets/__init__.py

```diff
@@ -43,8 +42,7 @@
     "ConcatDataset",
     "wikitext_dataset",
     "PreferenceDataset",
-    "the_cauldron_dataset",
     "SFTDataset",
     "hh_rlhf_helpful_dataset",
-    "llava_instruct_dataset",
+    "multimodal",
```

Comment:
Why not just expose the APIs directly in datasets/multimodal/__init__.py? Feels a bit weird to list a folder as a public API like this

Reply:
I tried not including multimodal in this init and the import path stopped working